lte/protos/session_manager.proto

```diff
@@ -789,6 +837,19 @@ message RatSpecificContextAccess {
     M5GSMSessionContextAccess m5g_session_context_rsp = 2;
   //}
 }
+message SessionAmbr {
+ enum AmbrUnit {
+   Kbps_0 = 0;
+   Kbps_1 = 1;
+   Kbps_4 = 2;
+   Kbps_16 = 3;
+   Kbps_64 = 4;
+ }
+ AmbrUnit downlink_unit_type =1;
+ uint32   downlink_units =2;   // Only to use lower 2 bytes (16 bit values)
+ AmbrUnit uplink_unit_type =3;
+ uint32   uplink_units =4;   // Only to use lower 2 bytes (16 bit values) 
+}
```

Comment:
no need to define a new AMBR message.

Reply:
Both are related we clubbed in one message, AMF had coded in couple of places, Is it hampering anyway ?

Reply:
It is not about whether it is hampering or not. Let's not inflate protos with new objects. The only difference I see is the AmbrUnit, which does not really add much value at this point.

Reply:
Removed "message SessionAmbr"